lib/ret_web/controllers/page_controller.ex

```diff
@@ -96,6 +96,9 @@ defmodule RetWeb.PageController do
 
   def render_for_path("/hub.service.js", _params, conn), do: conn |> render_page("hub.service.js")
 
+  def render_for_path("/app-config-schema.toml", _params, conn),
```

Comment:
i think we should expose these as `/hubs/schema.toml` and `/spoke/schema.toml`, and they can be served up simliar to the web manifest as a whitelisted file.

the fields can have categories similar to the current ita ones and be presented that way in the admin console

Reply:
i do agree conceptually that we serve up the full "app config" which includes spoke and hubs as you are doing now though

Reply:
ignore my second comment, didn't think it through (and we talked about what i think is ideal)